use-ssh/cleanup.ts

```diff
@@ -3,7 +3,11 @@ import { exec } from "@actions/exec";
 
 async function run() {
   try {
-    await exec("ssh-agent -k");
```

Comment:
The `ssh-agent` is shared when running `use-ssh` multiple times in the same job. 
When running `ssh-agent -k` it kills the agent for all the actions. 
Therefore, only the first action to run the `post` action script would complete with success. All of the others would fail with the following error:
<img width="789" alt="image" src="https://user-images.githubusercontent.com/39057274/156457006-f9c98ae1-01db-44fa-a930-2324f03b1b9a.png">